doc/client_lua_api.md

```diff
@@ -754,6 +759,20 @@ Call these functions only at load time!
     * returns reference to mod private `StorageRef`
     * must be called during mod load time
 
+### Mod channels
+* `minetest.mod_channel_join(channel_name)`
+	* Client joins channel `channel_name`. If channel has no member it's created. You 
+	  should listen from incoming messages with `minetest.register_on_modchannel_message`
+	  call to receive incoming messages. Warning, this function is asynchronous,
+	  you should use a minetest.after(2, function() ... end) at least, to permit server
```

Comment:
Warning belongs to a new list point and `minetest.after` into code formatters -> \`   \`